pkg/config/cli.go

```diff
@@ -94,6 +95,7 @@ func GetPersistentFlagSet() *pflag.FlagSet {
 	flagset.StringVarP(&CfgFile, "config", "c", "", "config file, use '-' to read the config from stdin")
 	flagset.BoolVarP(&Debug, "debug", "d", false, "Debug logging (default: false)")
 	flagset.StringVar(&DataDir, "data-dir", "", "Data Directory for k0s (default: /var/lib/k0s). DO NOT CHANGE for an existing setup, things will break!")
+	flagset.StringVar(&StatusSocket, "status-socket", "status.sock", "Name of the status socket file in the runDir.")
```

Comment:
I'd prefer this to be the full path, not the "file in the runDir".

Reply:
That would mean that we will allow users to put the file anywhere. This is not very practical from the perspective of `reset`.